src/components/landscape-sidebar/geo-description-widget/geo-description-widget-component.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { Loading } from 'he-components';
-import ShareModal from 'components/share-modal';
 import { VIEW_MODE } from  'constants/google-analytics-constants';
 import styles from './geo-description-widget-styles.module.scss';
+import * as actions from 'actions/url-actions';
+import { connect } from 'react-redux';
+import FixedHeader from 'components/fixed-header'
+
+const GeoDescriptionWidget = (props) => {
+  const { data, loading, error, view, changeGlobe } = props;
+
+  const handleBackClick = () => {
+    const params = { zoom: 7 }
+    view.goTo(params).then(() => changeGlobe(params));
```

Comment:
We do not need to change the `zoom` param on the `URL` after the `goTo` method has been resolved. We are already updating the `zoom` param whenever a change on the `view.zoom` happens through a watcher on the `zoom-widget` component. So we do not really need the `changeGlobe` action here.